src/OpenIddict.Core/OpenIddictProvider.Authentication.cs

```diff
@@ -180,12 +202,17 @@ public partial class OpenIddictProvider<TUser, TApplication> : OpenIdConnectServ
             ticket.SetResources(context.Request.GetResources());
             ticket.SetScopes(context.Request.GetScopes());
 
+            logger.LogDebug("AuthenticationTicket generated succesfully.");
+
             // Call SignInAsync to create and return a new OpenID Connect response containing the serialized code/tokens.
             await context.HttpContext.Authentication.SignInAsync(ticket.AuthenticationScheme, ticket.Principal, ticket.Properties);
 
+            logger.LogDebug("SignInAsync called.");
```

Comment:
Yes, but that would be great if we could avoid introducing meaningless messages like the ASP.NET team did for the OIDC client middleware :smile: 

What about something like `logger.LogInformation("The user session was successfully refreshed and a new access token was returned to the client application.");`? Maybe with some more details...